tests/Aspire.Hosting.Containers.Tests/WithDockerfileTests.cs

```diff
@@ -160,8 +160,8 @@ public async Task WithDockerfileResultsInBuildAttributeBeingAddedToManifest()
         });
         builder.Services.AddLogging(b => b.AddXunit(testOutputHelper));
 
-        var parameter = builder.AddParameter("message");
         builder.Configuration["Parameters:message"] = "hello";
+        var parameter = builder.AddParameter("message");
```

Comment:
Why can't it be done with a single call like in several other files?

Reply:
It could, but I left the usages in the "core/base API" tests to use the configuration for test coverage. Where for the hosting integrations that aren't part of the "core/base API" tests I changed to be simple, since these are the ones that are typically cloned to other hosting tests.